Marlin/src/pins/stm32f4/pins_BTT_SKR_PRO_common.h

```diff
@@ -40,75 +40,75 @@
 //
 // Servos
 //
-#define SERVO0_PIN                          PA1
+#define SERVO0_PIN                          PA1   // e.g. BLTOUCH (Marlin defaults the BLTouch control to the first servo pin)
 
 //
 // Trinamic Stallguard pins
 //
 #define X_DIAG_PIN                          PB10  // X-
 #define Y_DIAG_PIN                          PE12  // Y-
 #define Z_DIAG_PIN                          PG8   // Z-
-#define E0_DIAG_PIN                         PE15  // E0
-#define E1_DIAG_PIN                         PE10  // E1
-#define E2_DIAG_PIN                         PG5   // E2
+#define E0_DIAG_PIN                         PE15  // E0 (X+)
+#define E1_DIAG_PIN                         PE10  // E1 (Y+)
+#define E2_DIAG_PIN                         PG5   // E2 (Z+)
 
```

Comment:
There are no X/Y/Z max endstops on the pro so this just adds confusion.

Reply:
True, while it's silkscreened as E0/E1/E2, they are colloquially used as X+/Y+/Z+. 
See the `Limit Switches` section below. E.g. specifically:
```
#ifdef X_STALL_SENSITIVITY
 ...
#else 
  #define X_MIN_PIN                         PB10
  #define X_MAX_PIN                        PE15
```

Reply:
In addition, it's possible that someone will use sensorless homing along with E0/E1/E2 as X+/Y+/Z+ with physical endstops (for whatever reason).

Reply:
I know (mostly) any pin can be remapped for another feature, but it's confusing to keep track of in pins since we're not following the silkscreened labels. 🤷‍♂️

Reply:
Yeah, that's why I didn't remove the E0/E1/E2 designation. But without this comment, it's confusing from a user's perspective on why E0/E1/E2 is used as X/Y/Z_MAX_PIN later on in the file.

Reply:
> True, while it's silkscreened as E0/E1/E2, they are colloquially used as X+/Y+/Z+.

Comments on pins should indicate what the silkscreen says, when possible.

> without this comment, it's confusing from a user's perspective on why E0/E1/E2 is used as X/Y/Z_MAX_PIN later on in the file

When they are used this way they are actually disconnected from the `En_DIAG_PIN` on the stepper drivers. And then they may be used in any desired manner, not just for endstops.